packages/camera/camera_android_camerax/lib/src/process_camera_provider.dart

```diff
@@ -69,22 +91,73 @@ class ProcessCameraProviderHostApiImpl extends ProcessCameraProviderHostApi {
         as ProcessCameraProvider;
   }
 
-  /// Retrives the list of CameraInfos corresponding to the available cameras.
-  Future<List<CameraInfo>> getAvailableCameraInfosFromInstances(
-      ProcessCameraProvider instance) async {
+  /// Gets identifier that the [instanceManager] has set for
+  /// the [ProcessCameraProvider] instance.
+  int getProcessCameraProviderIdentifier(ProcessCameraProvider instance) {
     int? identifier = instanceManager.getIdentifier(instance);
     identifier ??= instanceManager.addDartCreatedInstance(instance,
```

Comment:
Why does this add the instance to the `InstanceManager` if it doesn't contain the `ProcessCameraProvider`.?Shouldn't `getInstancefromInstances` add the `ProcessCameraProvider` to the `InstanceManager`? This should throw an error if the `InstanceManager` doesn't contain `instance`.

Reply:
I agree. Fixed that.